safari/Info.plist

```diff
@@ -60,6 +60,7 @@
 				<string>miiverse.nintendo.net</string>
 				<string>api.gyazo.com</string>
 				<string>codepen.io</string>
+				<string>superstack.io</string>
```

Comment:
Usually if we need to add more permissions to Safari we also need to add them for Chrome/Edge. Are you sure that it won't work without this?